modules/appeals_api/app/swagger/higher_level_reviews/v0/swagger_dev.json

```diff
@@ -34,7 +49,6 @@
             "name": "X-VA-SSN",
             "required": true,
             "description": "Veteran's SSN",
-            "example": "706547821",
```

Comment:
I think we want to keep these examples, as they were recently updated in @michelpmcdonald's ticket to update on LOAST findings.

Reply:
It looks like the missing example values were previously just added to the JSON files (as opposed to the rswag specs themselves), so they all got overwritten once the JSON was regenerated - I think I've found a solution but I left a few comments with notes.